cli.js

```diff
@@ -142,7 +142,7 @@ function run(data) {
             if (err) {
                 error(err);
             } else {
-                process.stdout.write(val);
+                process.stdout.write(val, process.exit);
```

Comment:
Nothing in particular. I just don't like the fact that this change is
unrelated to the rest. You could make a new PR for this and keep things
clean.
On Jun 11, 2016 23:49, "Ben Zörb" <notifications@github.com> wrote:

> In cli.js
> <https://github.com/addyosmani/critical/pull/129#discussion_r66711590>:
>
> > @@ -142,7 +142,7 @@ function run(data) {
> >              if (err) {
> >                  error(err);
> >              } else {
> > -                process.stdout.write(val);
> > +                process.stdout.write(val, process.exit);
>
> I'm sure there was some reason i put it there even if i can't get it right
> now as this doesn't have anything todo with the PRs main purpose. Do you
> have any insights on this being a bad practice?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/addyosmani/critical/pull/129/files/1d124336fb5352b7c6734f669bb4f713b0a29f66#r66711590>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVVtYNsNnoz5U2xeSvk7hUqQxOh-tvdks5qKx9lgaJpZM4IJ0Vx>
> .
>